coffeebuddy/route_tables.py

```diff
@@ -1,13 +1,29 @@
 import datetime
+import itertools
+import logging
 
 import flask
+import webexteamssdk
 
 from coffeebuddy.model import Drink, User, Pay
 
 
 def init():
     @flask.current_app.route("/tables.html")
     def tables():
+        api = webexteamssdk.WebexTeamsAPI(access_token=flask.current_app.config["WEBEX_ACCESS_TOKEN"])
+        coffebuddyEmail = api.people.me().emails[0]
+
+        icon_out = '<i class="fa-solid fa-right-from-bracket"></i>'
```

Comment:
Do not mix Html and Python. Pass just `"in"` and `"out"` and use a table renderer in tables.html:98:

```js
{
    title: 'Direction', data: 'direction', render: (data, type, row) => {
        if (data == "in")
            return '<i class="fa-solid fa-right-from-bracket"></i>';
        else
            return '<i class="fa-solid fa-right-to-bracket color-berry"></i>';
    }
},
```

Reply:
You swapped them (I assume accidentally). Other than that, I applied that change.